src/base_station/log.cpp

```diff
@@ -10,25 +10,33 @@ constexpr int LOG_BUFFER_SIZE = 1024 * 1024;
 
 Handler handlers[MAX_HANDLERS];
 int num_handlers = 0;
+bool debugMode = true;
 	
 void register_handler(Handler handler) {
 	assert(num_handlers + 1 < MAX_HANDLERS);
 
 	handlers[num_handlers++] = handler;
 }
 
+bool toggleDebugMode() {
+	debugMode = !debugMode;
+	return debugMode;
+}
+
 void log(Level level, const char* format, ...) {
-	va_list list;
-	va_start(list, format);
+	if(debugMode) {
```

Comment:
This prevents any log messages from being printed when debugMode is false. Instead, we want to print anything that does not have Level::DEBUG.